### PR TITLE
Added schema for PPI ontology and MINT-PPI. Added Sam's CompoundSchem…

### PR DESCRIPTION
**Pull request by mengzhensun**
…a, CompoundSchemaEnum, GenomeAnnotation.

### CODE REVIEW

File: schema/BioOntologySchema.mcf [hunk 1]
```diff
@@ -0,0 +1,5206 @@
...
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "InteractionSourceEnum"
+description: "The interaction source database enumeration in biomedical ontologies."
+
+Node: dcid:AffinityChromatographyTechnology
+typeOf: dcs:InteractionTypeEnum
+name: "AffinityChromatographyTechnology"
+psimiID: "MI:0004"
+description: "This class of approaches is characterised by the use of affinity resins as tools to purify molecule of interest (baits) and their binding partners. The baits can be captured by a variety of high affinity ligands linked to a resin - for example, antibodies specific for the bait itself, antibodies for specific tags engineered to be expressed as part of the bait or other high affinity binders such as glutathione resins for GST fusion proteins, metal resins for histidine-tagged proteins"
```
Comment (pradh) [inline schema/BioOntologySchema.mcf:46]: For these descriptions, if you've written them yourself, this is good.

If these are copy/pastes from a source, then we add the URL string as value for the "descriptionUrl" property in this node.

File: schema/BioOntologySchema.mcf [hunk 2]
```diff
@@ -0,0 +1,5206 @@
+Node: dcid:psimiID
```
Comment (pradh) [inline schema/BioOntologySchema.mcf:1]: I'm assuming this is generated?

Have a comment at the top saying this is generated from a script with a <LINK-TO-SCRIPT>.

File: schema/BioOntologySchema.mcf [hunk 3]
```diff
@@ -0,0 +1,5206 @@
...
+typeOf: schema:Property
+name: "psimiID"
+description: "The identifier for Proteomics Standards Initiative Molecular Interaction."
+rangeIncludes: schema:Text
+domainIncludes:dcs:InteractionTypeEnum,dcs:InteractionDetectionMethodEnum,dcs:InteractionSourceEnum
+
```
Comment (pradh) [inline schema/BioOntologySchema.mcf:7]: BioMINTSchema.mcf file mixes both schema and data MCF. The data MCF should not be checked into github, should not be checked in here. Can we separate them?

For example:


Node: dcid:AuthorScore
typeOf: ConfidenceScoreTypeEnum
name: "AuthorScore"
description: "The confidence score of a protein-protein interaction given by author."

Node: dcid:IntActMiScore
typeOf: ConfidenceScoreTypeEnum
name: "IntactMiScore"
description: "The confidence score of a protein-protein interaction given in IntAct Molecular Interaction Database."

Node: dcid:bio/RPN1_YEAST_RPN3_YEAST
typeOf: ProteinProteinInteraction
name: "RPN1_YEAST_RPN3_YEAST"
interactingProtein: dcs:bio/UniProt_RPN1_YEAST,dcs:bio/UniProt_RPN3_YEAST
interactionDetectionMethod: dcs:TandemAffinityPurification
interactionType: dcs:PhysicalAssociation
interactionSource: dcs:Mint
intActID: "EBI-6941860"
mintID: "MINT-1984371"
imexID: "IM-15332-8532"
confidenceScore: [0.76 dcs:IntactMiScore]
pubMedID: "16554755"
imexID: "IM-15332"
mintID: "MINT-5218454"

File: schema/CompoundSchema.mcf [hunk 4]
```diff
@@ -0,0 +1,222 @@
+Node: dcid:ChemicalCompound
+name: "ChemicalCompound"
+typeOf: schema:Class
+subClassOf: schema:Thing
+description: "Chemical compound."
+sameAs: "https://www.ebi.ac.uk/chembl/"
```
Comment (pradh) [inline schema/CompoundSchema.mcf:6]: From schema.org/sameAs: "URL of a reference Web page that unambiguously indicates the item's identity. E.g. the URL of the item's Wikipedia page, Wikidata entry, or official website."

Given that, I'm puzzled by multiple nodes having the same sameAs in this file. Either we should provide uniquely identifying URLS, or drop sameAs, or change it to url (if that describes this).

File: schema/BioOntologySchema.mcf (hunk 1 above)
Comment (mengzhensun) [reply schema/BioOntologySchema.mcf:46]: The descriptions are directly parsed from the database file itself. Should I add descriptionUrl: "https://www.ebi.ac.uk/ols/ontologies/mi" to all the nodes in this schema? The corresponding piece in the database file is as below:
[Term]
id: MI:0004
name: affinity chromatography technology
def: "This class of approaches is characterised by the use of affinity resins as tools to purify molecule of interest (baits) and their binding partners. The baits can be captured by a variety of high affinity ligands linked to a resin - for example, antibodies specific for the bait itself, antibodies for specific tags engineered to be expressed as part of the bait or other high affinity binders such as glutathione resins for GST fusion proteins, metal resins for histidine-tagged proteins." [PMID:7708014]
subset: PSI-MI_slim
synonym: "affinity chrom" EXACT PSI-MI-short []
synonym: "Affinity purification" EXACT PSI-MI-alternate []
is_a: MI:0091 ! chromatography technology
is_a: MI:0400 ! affinity technology

File: schema/CompoundSchema.mcf (hunk 4 above)
Comment (mengzhensun) [reply schema/CompoundSchema.mcf:6]: Sam completed this schema. We will discuss this with her in the meeting.

File: schema/BioOntologySchema.mcf (hunk 1 above)
Comment (pradh) [reply schema/BioOntologySchema.mcf:46]: description: "This class of approaches is characterised by the use of affinity resins as tools to purify molecule of interest..."
descriptionUrl: "https://www.ebi.ac.uk/ols/ontologies/mi"

File: schema/CompoundSchema.mcf (hunk 4 above)
Comment (pradh) [reply schema/CompoundSchema.mcf:6]: Per morning discussion are we changing this to "url" ?

**Pull request merged**